style.css

```diff
@@ -309,7 +341,7 @@ label {
     align-content: center;
   }
 
-  #nav_menu {
+  #nav_list {
```

Comment:
[OPTIONAL] It is not advisable to use IDs as CSS selectors because if another element in the page uses the same/similar style, you would have to write the same CSS again. Even if you don't have more than one element with that style right now, it might come later. Hence it is always advisable to use classes as CSS selectors. So my advise to you is to always use classes as CSS selectors and  not Ids.

Reply:
> [OPTIONAL] It is not advisable to use IDs as CSS selectors because if another element in the page uses the same/similar style, you would have to write the same CSS again. Even if you don't have more than one element with that style right now, it might come later. Hence it is always advisable to use classes as CSS selectors. So my advise to you is to always use classes as CSS selectors and not Ids.

- [OPTIONAL] Kindly address this 👍🏻